doc/source/language/flow.rst

```diff
@@ -87,7 +87,9 @@ Every time you read the value of the "variable", it executes the expression
 again.
 
 .. note::
-    If a ``LOCK`` expression is used with a flight control such as ``THROTTLE`` or ``STEERING``, then it will get continually evaluated in the background :ref:`each physics tick <cpu hardware>`.
+    If a ``LOCK`` expression is used with a flight control such as ``THROTTLE`` or ``STEERING``, then it will get continually evaluated in the background :ref:`each physics tick <physics tick>`.
```

Comment:
I would change the phrasing from "continually" to "repeatedly", and given the documentation for triggers specifically says that they do not run in the "background" it might be good to find another way to phrase the last part of this line.